recipe/meta.yaml

```
@@ -12,25 +12,20 @@ source:
   sha256: {{ sha256sum }}
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
-requirements:
+requirements:  
+  build:
+    - {{ compiler('cxx') }}
   host:
     - python
     - setuptools
+    - setuptools_scm
     - wheel
     - pip
```

```suggestion
    - pip
    - packaging
```
`packaging` is required to build a package, see:
- https://github.com/wmayner/pyemd/blob/1e11dc328b77fc34f9a2546ab6afe2b91d88ca83/setup.py#L87
- https://github.com/wmayner/pyemd/blob/1e11dc328b77fc34f9a2546ab6afe2b91d88ca83/setup.py#L12
- https://github.com/wmayner/pyemd/releases